joomla/joomla-cms
# [4.0] Created a HTMLHelper to render an icon

Pull Request for Issue # .

### Summary of Changes

This PR will make it possible to use a HTMLHelper to place an icon. 
Just like there is a HTMLHelper to place an anchor and an image. 
- anchor => https://github.com/joomla/joomla-cms/blob/b2e4bf0bd26ab2bb718148459284242bcce8bc56/libraries/src/HTML/HTMLHelper.php#L253
- image => https://github.com/joomla/joomla-cms/blob/b2e4bf0bd26ab2bb718148459284242bcce8bc56/libraries/src/HTML/HTMLHelper.php#L574

### Testing Instructions

- Joomla 4 Administrator
- See the quick icons on the dashboard
- Apply patch
- See the quick icons on the dashboard ... it is still the same. Only to be implemented by the HTMLHelper now

### Actual result BEFORE applying this Pull Request

placing an icon would be: 

```
<span class="icon-joomla" aria-hidden="true"></span>
<span class="icon-joomla icon-white" aria-hidden="true"></span>
<span class="icon-joomla icon-white" aria-hidden="true" tabindex="0" title="Hello World"></span>
icon-joomla
<span class="fab fa-joomla" aria-hidden="true"></span>
```
### Expected result AFTER applying this Pull Request

```
echo HTMLHelper::_('icons.icon', 'joomla', TEXT::_('HELLO_WORLD'));
```

### Documentation Changes Required

most likely...
